src/commands.ts

```diff
@@ -59,10 +59,8 @@ export async function authorizeOrg(orgName?: string) {
 		title: "Hutte: Setting Org as Default & Switching to the Org's Git Branch",
 		cancellable: false
 	}, () => {
-		try {
-			// const output = commandSync(String.raw`echo -n "${orgName}" | sfdx hutte:org:authorize --no-pull`, { shell:true,  cwd: getRootPath() });
-			// TODO: Add orgName as a parameter to sfdx hutte:org:authorize in Hutte CLI and refactor this to not use a Unix Shell and therefore make it compatible with more devices.
-			const output = execSync(String.raw`echo -n "${orgName}" | sfdx hutte:org:authorize --no-pull`, { cwd: getRootPath(), shell: "/bin/bash" }).toString();
+		try {			
+			execSync(String.raw`sf hutte org authorize --no-pull --org-name ${orgName}`, { cwd: getRootPath() }).toString();
```

Comment:
- Is there a reason why you have chosen `child_process.execSync` over `execa.commandSync`?
- Do we need this `String.raw`?
- I'm wondering if we need to quote the `${orgName}` in case the org name contains whitespace. Is execa maybe doing this automatically, or am I misunderstanding this?
  > Arguments are automatically escaped ([execaCommand](https://github.com/sindresorhus/execa#execacommandcommand-options))